pkg/networkservice/common/translation/client.go

```diff
@@ -0,0 +1,72 @@
+// Copyright (c) 2020 Doc.ai and/or its affiliates.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at:
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package translation provides client chain element to translate server request mechanisms to client request mechanisms
+// and client connection mechanisms to server connection mechanisms
+package translation
```

Comment:
Can we rename this to something meaning that it is related to mechanisms?

For example: `translationmechanism` or something similar.

Reply:
Renamed to `mechanismtranslation`.